templates/chrony.conf.j2

```diff
@@ -21,6 +26,17 @@ makestep {{ timesync_step_threshold if timesync_step_threshold > 0.0 else '1.0'
 # Enable kernel synchronization of the real-time clock (RTC).
 rtcsync
 
+{% if __timesync_chrony_version is version('3.0', '>=') and
+        timesync_ntp_hwts_interfaces|length > 0 %}
+# Enable hardware timestamping.
+{% for interface in timesync_ntp_hwts_interfaces %}
+hwtimestamp {{ interface }}{% if __timesync_chrony_version is version('3.1', '>=') %}
+ minpoll {{ [ timesync_ntp_servers | map(attribute='minpoll', default=6) | min, 0] | min
```

Comment:
https://fedorapeople.org/groups/linuxsystemroles/logs/linux-system-roles-timesync-pull-linux-system-roles_timesync-88-ebd5733-rhel-8-20201215-154341/artifacts/ansible.log
```
TASK [linux-system-roles.timesync : Generate chrony.conf file] *****************
task path: /tmp/tmp0blwj5i4/tasks/main.yml:106
fatal: [/cache/rhel-8.qcow2]: FAILED! => {"changed": false, "msg": "FilterArgumentError: Unexpected keyword argument 'default'"}
```
I guess `map` in jinja2.7 does not support the `default` parameter - https://github.com/pallets/jinja/blob/2.7.3/jinja2/filters.py#L798

https://github.com/pallets/jinja/blob/2.7.3/docs/templates.rst

Reply:
Ok. I'm trying to replace it with the rejectattr filter.